CHANGELOG.md

```diff
@@ -0,0 +1,20 @@
+# push-to-registry Changelog
+
+## v2
+- Rename `tag` input to `tags`, and allow you to push multiple tags of the same image
+- Add input parameter `extra_args` to append extra args to the podman push
+- Rename `registry-path` output parameter to `registry-paths`, and allow you to output multiple image registry paths of the pushed image
```

Comment:
Rename `registry-path` output to `registry-paths`, which is a JSON-parseable array containing all registry paths of the pushed image. The size of the output array is the number of `tags` that were pushed.